File: FeedleDataTier/FeedleDataTier/Network/RespondToFriendResponse.cs
using System.Collections.Generic;
using System.Text.Json.Serialization;
using FeedleDataTier.Models;

namespace FeedleDataTier.Network
{
    public class RespondToFriendResponse : Request
    {
        [JsonPropertyName("userFriends")]
        public List<UserFriend> UserFriends { get; set; }
        public RespondToFriendResponse(List<UserFriend> userFriends) : base(RequestType.RespondToFriendResponse)
        {
            UserFriends = userFriends;
        }
    }
}

File: Backend/src/main/java/com/feedle/feedleapi/Networking/RespondToFriendResponse.java
package com.feedle.feedleapi.Networking;

import com.fasterxml.jackson.annotation.JsonIgnore;
import com.feedle.feedleapi.Models.UserFriend;

import java.util.ArrayList;

public class RespondToFriendResponse extends Request {
    private ArrayList<UserFriend> userFriends;
    public RespondToFriendResponse(ArrayList<UserFriend> userFriends)
    {
        super(RequestType.ResponseToFriendResponse);
        this.userFriends = userFriends;
    }

    @JsonIgnore
    public ArrayList<UserFriend> getUserFriends() {
        return userFriends;
    }
}
